Add radiotherm CT80 current humidity support

zombielinux opened the pull request:
## Description:

Added a check for if device is a CT80, and if so, queries the humidity object to get the current measured humidity reading.

**Pull request with documentation for [home-assistant.io](https://github.com/home-assistant/home-assistant.io) (if applicable):** home-assistant/home-assistant.io#<home-assistant.io PR number goes here>

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [x] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [x] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [x] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

zombielinux committed:
Added CT80 Current Humidity Support

Added a check for if device is a CT80, and if so, queries the humidity object to get the current measured humidity reading.

zombielinux committed:
Update climate.py

Removed whitespace on line 229

elupus reviewed (changes requested):
This can't be working. You don't actually add the humidity property anywhere.

zombielinux committed:
Update climate.py

Added humidity property. Version on local machine had that from previous tinkering.

zombielinux commented:
@elupus Good catch. I had that on my local machine, didn't get that copied over.

zombielinux committed:
Update climate.py

Removed whitespace

zombielinux committed:
Update climate.py

Fixed tstat error handling for humidity data.

MartinHjelmare commented:
Can be merged when build passes.
